composable.yml

```diff
@@ -7,6 +7,17 @@
         infrared_oc_deploy_template:  "{{ ansible_user_dir }}/.infrared/plugins/tripleo-overcloud/templates/overcloud_deploy.sh.j2"
       shell: |
         cp   {{ oc_deploy_template }} {{ infrared_oc_deploy_template }}
+
+    - name: avoid compute role of controller machine type
```

Comment:
If I understand correctly, this task will only run when the number of nodes of this hardware type is equal to the expected number of controllers? If yes, then maybe the task name can be changed accordingly to avoid confusion?